### PR TITLE
puddletag: Updated resource versions, checksums and added a PATH variable to starter script.

### PR DESCRIPTION
**Pull request by chincheta0815**
add path '/usr/local/bin' to starter script. launchtl setenv... has no effect.

### CODE REVIEW

File: Library/Formula/puddletag.rb [hunk 1]
```diff
@@ -48,7 +49,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", HOMEBREW_PREFIX/"lib/python2.7/site-packages"
 
     bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PATH => "/usr/local/bin", :PYTHONPATH => ENV["PYTHONPATH"])
```
Comment (DomT4) [inline Library/Formula/puddletag.rb]: Don't hardcode `/usr/local`. Not everyone has Homebrew installed there. It should ideally be `"#{bin}"` or if absolutely necessary `"#{HOMEBREW_PREFIX}/bin"`

**Comment by chincheta0815** [comment]
Is the line okay this way now?

**Comment by chincheta0815** [comment]
I don't want to be nasty, but is this commit okay now?

**Comment by chincheta0815** [comment]
Added some component updates to the revision.

**Comment by dunn** [comment]
Sorry, why is adding `PATH` to the wrapper scripts necessary?

**Comment by chincheta0815** [comment]
That puzzles me also a litte...
Puddletag has a tagsource feature called "AcoutId" (can be seen in the Tag Sources Panel). This feature requires the bin "fpcalc" in PATH which is part of the optional package chromaprint.

When starting puddletag from the console everything is okay. Starting it with the .app bundle fpalc is not found. When it is put into PATH it works.

The 'fpcalc' thing is coded in pyacoustid which comes with puddletag and is based on pypython and adopted by upstream (https://github.com/keithgg/puddletag/blob/master/source/puddlestuff/tagsources/_acoustid.py; line 43) and a check for it is done (https://github.com/keithgg/puddletag/blob/master/source/puddlestuff/tagsources/acoust_id.py; line 385 to 386).

Maybe you have another idea off getting rid of PATH? Actually I do not like it also very much, but some things like inreplace also seem not to be very nice...

**Comment by dunn** [comment]
Do you have `/usr/local/bin` in your own `PATH`?

**Comment by chincheta0815** [comment]
Talking about a clean, fresh opened shell in the console: yes
echo $PATH --> /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
And I did not tweak anything by .bash_profile etc...
I am on El Capitan. Maybe this is also something version specific which make it more complicated...

**Comment by dunn** [comment]
Oh, I missed the bit about the app bundle.  I guess it makes sense that `PATH` might not include `/usr/local/bin` in those cases.

Merged in 0fc6c4fd26c55a9541f63a8a4f5b204add644f7a, thank you!

**Comment by chincheta0815** [comment]
Thanks a lot for your efforts guys!

**Pull request closed without merging**